docs/contributing.rst

```diff
@@ -4,15 +4,16 @@ Contributing
 
 Help is always appreciated!
 
-To get started, you can try :ref:`building-from-source` in order to familiarize
-yourself with the components of Solidity and the build process. Also, it may be
-useful to become well-versed at writing smart-contracts in Solidity.
+To get started, you can try `building-from-source 
```

Comment:
It is [working](https://solidity.readthedocs.io/en/latest/contributing.html#contributing).  So, the URL should disappear and the old way of `:ref:` role should re-appear.